src/components/BoxColor.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import '../components/BoxColor.css'
+
+class BoxColor extends React.Component {
+    render() {
+

```

Comment:
As you can see in your terminal, you aren't using this variables under, you get them from the props. So you can delete them and it still works fine!